# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../tcp_wrappers/0027-tcp_wrappers-7.6-static-sccsid.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- a/workarounds.c
+++ b/workarounds.c
@@ -8,7 +8,7 @@
   */
 
 #ifndef lint
-char    sccsid[] = "@(#) workarounds.c 1.6 96/03/19 16:22:25";
+static char    sccsid[] = "@(#) workarounds.c 1.6 96/03/19 16:22:25";
 #endif
 
 #include <sys/types.h>
